### ISSUE TITLE
incorrect output from eq when characters certain characters used as input

### ISSUE THREAD

**Issue by margo0ka** [supabase-community/supabase-py#90]
[related issue](https://github.com/supabase-community/supabase-py/issues/37)

## **Describe the bug**
When you try to select rows where col1 = a.b it will look for "a.b" in the table (with quotes) and not a.b (without quotes) when you are using the eq function.

I have the table `test`
```
[{'id': 1, 'col1': 'a.b', 'col2': 'apple'}, 
{'id': 2, 'col1': '"a.b"', 'col2': 'banana'}, 
{'id': 3, 'col1': 'a,c', 'col2': 'mango'}, 
{'id': 4, 'col1': '"a,c"', 'col2': 'melon'}, 
{'id': 5, 'col1': 'aa', 'col2': 'cherry'}, 
{'id': 6, 'col1': '"aa"', 'col2': 'grape'}]
```
the command
```
x = supabase.table("test").select("*").eq("col1","a.b").execute()
print(x)
```
will get you
`{'data': [{'id': 2, 'col1': '"a.b"', 'col2': 'banana'}], 'status_code': 200}`

which is incorrect. You should get
`{'data': [{'id': 1, 'col1': 'a.b', 'col2': 'apple'}], 'status_code': 200}`

The same thing happens when you try with `a,c`. However `aa` works exactly as intended.

```
x = supabase.table("test").select("*").eq("col1","aa").execute()
print(x)
```
`{'data': [{'id': 5, 'col1': 'aa', 'col2': 'cherry'}], 'status_code': 200}`

This seems like an issue with sanitizing the input. 

![s](https://user-images.githubusercontent.com/94395604/144296342-c0fd558e-c9ce-452a-b47a-9403e5e4947a.png)
[request_builder.py](https://github.com/supabase-community/postgrest-py/blob/2fc29b272323203bfa0b4f5f59ae12bde08dc530/postgrest_py/_async/request_builder.py)
![s2](https://user-images.githubusercontent.com/94395604/144296016-691637a0-04aa-4faf-aa26-ec1a094d2965.png)
[utils.py](https://github.com/supabase-community/postgrest-py/blob/2fc29b272323203bfa0b4f5f59ae12bde08dc530/postgrest_py/_async/request_builder.py)

The desired result can seemingly be achieved by bypassing eq and not using sanitizing_param.
```
x = supabase.table("test").select("*").filter("col1","eq","a.b").execute()
print(x)
```
`{'data': [{'id': 1, 'col1': 'a.b', 'col2': 'apple'}], 'status_code': 200}`

I am using `supabase-py` version `0.0.3`.

**Comment by leynier** [comment]
@margo0ka please, install `supabase` instead of `supabase-py` and test it. Thanks in advance.

**Comment by margo0ka** [comment]
@leynier ah sorry I think I should have said `supabase` instead of `supabase-py`. I have whatever `pip3 install supabase` gives you.

**Comment by margo0ka** [comment]
Excuse the ugly screenshot but I just tried it with javascript and it's a different result from using it on python. So the javascript one works as I would expect.

![image](https://user-images.githubusercontent.com/94395604/144338798-164b763c-1d40-4cca-b68b-e853a16efa37.png)

**Comment by leynier** [comment]
@margo0ka thanks for the report, I will check and try to solve the bug. Likewise, if you want to collaborate and try to solve it, it would be super good.

**Comment by alexmercerind** [comment]
Hey guys!
Thanks for the work on this library, I recently faced this issue aswell & started a discussion thinking I was doing something wrong. 

https://github.com/supabase-community/supabase-py/discussions/145#discussioncomment-2168399. I thought I should leave a link here just in case it helps reaching to a solution to this problem.

Thanks again.

**Comment by anand2312** [comment]
This has been fixed in [postgrest-py#108](https://github.com/supabase-community/postgrest-py/pull/108). Thanks for the report!
